.travis.yml

```diff
@@ -2,8 +2,10 @@ language: node_js
 before_install:
  - sudo apt-get install build-essential
  - npm i mocha -g
+ - npm i maboiteaspam/cozy-light#refactoring -g
 node_js:
-    - "0.10"
+    - "0.10.34"
+    - "0.10.35"
```

Comment:
We should use a version compatible with the Raspberry Pi. Most users runs Cozy Light on a Raspberry Pi.